exec/java-exec/src/main/resources/drill-module.conf

```diff
@@ -502,6 +502,8 @@ drill.exec.options: {
     store.format: "parquet",
     store.hive.optimize_scan_with_native_readers: false,
     store.json.all_text_mode: false,
+    store.json.writer.non_numeric_numbers: false,
+    store.json.reader.non_numeric_numbers: false,
```

Comment:
Have we tested all Drill's floating point methods to ensure that they correctly handle NaN and INF?

Reply:
No, we don't and we should test them. I think we have to create separate jira for testing math functions, because code changes from this PR doesn't affect logic of any math function, and while testing math function there will be other issues not connected with with functionality (like BigInteger constructor doesn't accept NaN, Infinity and others)

Reply:
OK. So, if the rest of Drill either does not support (or we don't know if it supports) NaN and Inf, should we introduce the change here that potentially leads to failures elsewhere?

Do we know if JDBC and ODBC support these values? (I suppose they do as they are features of Java's float/double primitives...)